eve/main.yml

```diff
@@ -130,3 +131,103 @@ stages:
             echo "%(secret:ssh_pub_keys)s" >> ~/.ssh/authorized_keys &&
             sleep 3600
           doStepIf: false
+
+  multiple-nodes:
+    worker:
+      type: openstack
```

Comment:
Do I get it correct this node is an OpenStack VM, but it's not used as the bootstrap node itself? If that's the case, can it either be the bootstrap node, or can we run this in a container?

Reply:
Yeah that's right.

In the `single-node` stage, the CI worker (an OpenStack VM) is used as the bootstrap node itself. There is no VM created on top of this worker.

In the new `multiple-nodes` stage, the CI worker (still an OpenStack VM) is used as some kind of bastion to instantiate a fresh environment of VMs with one bootstrap node and X (2 by default) additional nodes.

In this new stage, we are in the same situation as the developer on its laptop running `vagrant`. There is no mix between the source code, and the ISO downloaded on the bootstrap node.

I'd prefer that we stay this way in the `multiple-nodes` stage since I think we want to have a clearly distinct env on which we can really test a fresh install.